client/my-sites/sidebar/sidebar.jsx

```diff
@@ -258,28 +264,23 @@ module.exports = React.createClass( {
 
 	upgrades: function() {
 		var site = this.getSelectedSite(),
-			target = null,
```

Comment:
I don't remember making those changes, I guess this is due to a bad rebase.
Removing them!